src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/recommendation/RecommendationResponse.kt

```diff
@@ -82,6 +82,7 @@ data class RecommendationResponse(
   val releaseUnderECSL: Boolean? = null,
   val dateOfRelease: LocalDate? = null,
   val conditionalReleaseDate: LocalDate? = null,
+  val deleted: Boolean? = false,
```

Comment:
Can we make this non nullable?  If it's null, it probably has to be interpreted as false, so we should make that the case explicitly.

Reply:
done